src/components/Carousel/Carousel.scss

```diff
@@ -0,0 +1,25 @@
+.Carousel {
+  display: flex;
+  align-items: center;
+  .Carousel-listWrapper {
+    display: flex;
+    flex-direction: row;
+    position: relative;
+    overflow-x: hidden;
+  }
+  .Carousel-list {
+    display: flex;
+    flex-direction: row;
+    position: absolute;
```

Comment:
Could you describe what `position: absolute` is doing here? Just for clarity's sake.

Reply:
Absolute position is to make carousel animate on basis the left property.